src/odin/mods/mod.rs

```diff
@@ -157,7 +156,10 @@ impl ValheimMod {
     if let Ok(maybe_manifest) = self.try_parse_manifest(archive) {
       let name = maybe_manifest.name;
       let mod_dir = format!("{}/", name);
-      let mod_dir_exists = archive.file_names().any(|file_name| file_name == mod_dir);
+
+      let mod_dir_exists = archive
+        .file_names()
+        .any(|file_name| file_name.starts_with(&mod_dir));
```

Comment:
This looks like it changes the validation from matching the directory name exactly, to just looking for a prefix?

Reply:
indeed @lafrad,  the zip module used to list the directory independently but they removed that feature, So by matching the prefix of `<some dir>/` i was able to validate it will match the directory format

Reply:
Cool!  thanks for the explanation!